proposals/lip-0045.md

```diff
@@ -618,8 +618,8 @@ terminatedStateAccountSchema = {
 ```
 ##### Properties
 
-* `stateRoot`: The state root of the terminated chain, initialized to `chainAccount(chainID).lastCertificate.stateRoot`, where `chainID` is the chain ID of the terminated chain. If the account is not initialized, it is set to `EMPTY_HASH` instead.
-* `mainchainStateRoot`: The state root of the mainchain at the moment in which the chain was terminated, set to `chainAccount(getMainchainID()).lastCertificate.stateRoot` for non-initialized terminated state accounts. If the account is initialized, it is set to `EMPTY_HASH` instead.
+* `stateRoot`: The state root of the terminated chain, initialized to `chainAccount(chainID).lastCertificate.stateRoot`, where `chainID` is the chain ID of the terminated chain, or to the state root given in the parameters of the sidechain terminated CCM. If the account is not initialized, it is set to `EMPTY_HASH` instead.
```

Comment:
Suggestion to make this cleaner: 

```suggestion
* `stateRoot`: The state root of the terminated chain. In case there is a chain account of the terminated chain, it is initialized to `chainAccount(chainID).lastCertificate.stateRoot`, where `chainID` is the chain ID of the terminated chain. If an account for terminated chain does not exist (this can happen only in sidechains) it is initialized to the state root given in the parameters of the sidechain terminated CCM. If the account is not initialized, it is set to `EMPTY_HASH` instead.
```

Reply:
I rephrased it a bit. I don't think we need to give the full fledged details again here

Reply:
The idea was to make it cleaner that the second case can happen only if there is no direct channel. 

Fine to keep it as it is for now.